cts/lab/CIB.py

```diff
@@ -127,7 +127,7 @@ def get_node_id(self, node_name):
         (rc, output) = self.Factory.rsh(self.Factory.target,
             r"""awk -v RS="}" """
             r"""'/^(\s*nodelist\s*{)?\s*node\s*{.*(ring0_addr|name):\s*%s(\s+|$)/"""
-            r"""{gsub(/.*nodeid:\s*/,"");gsub(/\s+.*$/,"");print}'%s"""
+            r"""{gsub(/.*nodeid:\s*/,"");gsub(/\s+.*$/,"");print}' %s"""
```

Comment:
Doh. Thankfully this was only used in one place, with the only effect being a random selection of two fencing configuration options would always choose the same one.